src/webargs/flaskparser.py

```diff
@@ -118,5 +117,6 @@ def get_default_request(self):
 
 
 parser = FlaskParser()
+
```

Comment:
Empty line added through subsequent edits while working on this PR. Remove it and move the werkzeug import up and this file is actually untouched by the whole PR.

Reply:
I have to reread this PR a bit. I tried to revert any changes from the base PR which became irrelevant under my rewrite, but I definitely missed a couple.

Reply:
No pb. It's mostly cosmetic. I tend to spend too much time struggling with `git rebase -i` to even have clean intermediate commits, for History (although nobody will ever study _that_ History).